pkgs/misc/emulators/ppsspp/default.nix

```diff
@@ -14,9 +14,9 @@ stdenv.mkDerivation rec{
 
   src = fetchgit {
```

Comment:
I think PPSSPP needs to use fetchgit because the repo uses submodules and fetchFromGitHub currently [lacks submodule support](https://github.com/NixOS/nixpkgs/issues/15559).